rktl_control/nodes/velocity_controller

```diff
@@ -61,7 +61,7 @@ class VelocityControllerROS(object):
         self.steering_pub = rospy.Publisher('effort/steering', Float32, queue_size=1)
 
         # Subscribers
-        rospy.Subscriber('cmd_vel', Twist, self.command_cb)
+        rospy.Subscriber('cmd', Twist, self.command_cb)
```

Comment:
can we use `control` instead? This also shouldn't be a twist. We need to make a custom message for it.